webapp/src/ts/modules/messages/messages.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="filters">
+  <div class="inner">
+    <!--<mm-navigation></mm-navigation>-->
+  </div>
+</div>
+
+<div class="inner">
+  <div class="inbox page">
+
+    <div id="message-list" class="col-sm-4 inbox-items left-pane">
+      <ul *ngIf="!loading">
+        <li class="content-row"
+          *ngFor="let msg of conversations; trackBy: listTrackBy"
```

Comment:
I might need this or something similar to load by default the first conversation in the list. I was planning to add this in the next PR about loading content. :)

Reply:
I don't think you need it for loading the 1st message anymore. 
It _was_ used before because we relied on the view sort filter to sort messages 🤯, but now we sort messages before passing to the template . 
It might be used by something else? Unsure.